<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hospital DB Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="patientTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="PatientRecordWidget" name="recordsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="aOpenDB"/>
    <addaction name="aCloseDB"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="aAddPatient"/>
    <addaction name="aRemovePatient"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="aEnglish"/>
     <addaction name="aRussian"/>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aAbout"/>
    <addaction name="aAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string notr="true">toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aAddPatient"/>
   <addaction name="aRemovePatient"/>
   <addaction name="aAddRecord"/>
   <addaction name="aRemoveRecord"/>
   <addaction name="aAddRadiograph"/>
   <addaction name="aRemoveRadiograph"/>
   <addaction name="aNextRadiograph"/>
   <addaction name="aPrevRadiograph"/>
  </widget>
  <action name="aRemovePatient">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/remove_patient.png</normaloff>:/icons/remove_patient.png</iconset>
   </property>
   <property name="text">
    <string>Remove patient</string>
   </property>
   <property name="toolTip">
    <string>Remove patient from DataBase</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+D</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
  </action>
  <action name="aAddPatient">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/add_patient.png</normaloff>:/icons/add_patient.png</iconset>
   </property>
   <property name="text">
    <string>Add patient</string>
   </property>
   <property name="toolTip">
    <string>Add new patient to DataBase</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
  </action>
  <action name="aOpenDB">
   <property name="text">
    <string>Open DataBase</string>
   </property>
  </action>
  <action name="aCloseDB">
   <property name="text">
    <string>Close DataBase</string>
   </property>
   <property name="toolTip">
    <string>Close connection to DataBase</string>
   </property>
  </action>
  <action name="aEnglish">
   <property name="text">
    <string>English</string>
   </property>
   <property name="toolTip">
    <string notr="true">en</string>
   </property>
  </action>
  <action name="aRussian">
   <property name="text">
    <string>Russian</string>
   </property>
   <property name="toolTip">
    <string notr="true">ru</string>
   </property>
  </action>
  <action name="aDisplayRecords">
   <property name="text">
    <string>Display Records</string>
   </property>
  </action>
  <action name="aAddRecord">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/add_record.png</normaloff>:/icons/add_record.png</iconset>
   </property>
   <property name="text">
    <string>Add record</string>
   </property>
  </action>
  <action name="aRemoveRecord">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/remove_record.png</normaloff>:/icons/remove_record.png</iconset>
   </property>
   <property name="text">
    <string>Remove record</string>
   </property>
  </action>
  <action name="aAddRadiograph">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/add_radiograph.png</normaloff>:/icons/add_radiograph.png</iconset>
   </property>
   <property name="text">
    <string>Add radiograph</string>
   </property>
  </action>
  <action name="aRemoveRadiograph">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/remove_radiograph.png</normaloff>:/icons/remove_radiograph.png</iconset>
   </property>
   <property name="text">
    <string>Remove radiograph</string>
   </property>
  </action>
  <action name="aDisplayRadiographs">
   <property name="text">
    <string>Display radiographs</string>
   </property>
  </action>
  <action name="aDisplayEverything">
   <property name="text">
    <string>Display everything</string>
   </property>
   <property name="toolTip">
    <string>Displays all rows in record and radiograph tables</string>
   </property>
  </action>
  <action name="aNextRadiograph">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/next_radiograph.png</normaloff>:/icons/next_radiograph.png</iconset>
   </property>
   <property name="text">
    <string>Next Radiograph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="aPrevRadiograph">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/previous_radiograph.png</normaloff>:/icons/previous_radiograph.png</iconset>
   </property>
   <property name="text">
    <string>Previous radiograph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="aAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="aAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PatientRecordWidget</class>
   <extends>QWidget</extends>
   <header>patientrecordwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
